src/exchanges.js

```diff
@@ -4,6 +4,9 @@ const promised = require('./promised.js');
 const { Wallet } = require('./wallet.js');
 const codesList = require('./codesList.json');
 
+const green = '\x1b[32m';
```

Comment:
Add colors to the object and exports from promised.js or use my library(advertising moment)

Reply:
Need to create new branch and Pull Request for this change

Reply:
Done in #59